python/tvm/auto_scheduler/search_task.py

```diff
@@ -227,6 +227,12 @@ def __init__(
         if isinstance(target_host, str):
             target_host = Target(target_host)
 
+        if layout_rewrite_option is None:
+            layout_rewrite_option = LayoutRewriteOption.NO_REWRITE
+            if target.kind.name == "llvm" or \
```

Comment:
1. move `enable_layout_rewrite` from `python/tvm/auto_scheduler/relay_integration.py` to this file.
2. call `enable_layout_rewrite` function here